packages/svelte/src/compiler/phases/2-analyze/validation.js

```diff
@@ -530,6 +531,19 @@ export const validation = {
 			}
 		}
 
+		if (interactive_elements.has(node.name)) {
```

Comment:
Should this use the more complete `disallowed_contents` from the util instead?

Reply:
I initially tried that., but it gets complicated, as you can nest some contents there (if you nest a table inside a cell of another table, this is valid).